jcommune-service/src/test/java/org/jtalks/jcommune/service/transactional/TransactionalAuthenticatorTest.java

```diff
@@ -486,6 +494,60 @@ public void userMustHaveInitialFieldValuesWhenPluginRegistrationFailWithRegistra
         assertReflectionEquals(expectedRegisterUserDto, registerUserDto);
     }
 
+    @Test
+    public void userMustSetToNullPasswordAndPasswordConfirmFieldsWhenTheyAreDifferent() throws Exception {
```

Comment:
`emptiesPassowordAndConfirmationFields_ifTheyDoNotMatch`

Maybe let's add some javadocs here explaining why it behaves like this.